### ISSUE TITLE
Unknown Crash! 

### ISSUE THREAD

**Issue by zombieknifer223** [Wurst-Imperium/Wurst-Client#1398]
When I was buying a bow and arrows from an Admin Shop, it accidentally activate BowAinbot and when it shot, it crashed Wurst. Here is the report

```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 7/27/15 1:05 PM
Description: Rendering entity in world

java.lang.ClassCastException: amj cannot be cast to java.lang.String
	at xv.e(SourceFile:108)
	at wv.k_(Entity.java:2622)
	at wv.d_(Entity.java:2362)
	at cpu.a(Render.java:377)
	at cqg.a(RenderItemFrame.java:228)
	at cqg.a(RenderItemFrame.java:83)
	at cqg.a(RenderItemFrame.java:255)
	at cpt.a(RenderManager.java:380)
	at cpt.a(RenderManager.java:337)
	at cpt.a(RenderManager.java:304)
	at ckn.a(RenderGlobal.java:654)
	at cji.a(EntityRenderer.java:1419)
	at cji.a(EntityRenderer.java:1333)
	at cji.b(EntityRenderer.java:1158)
	at bsu.as(Minecraft.java:1144)
	at bsu.a(Minecraft.java:421)
	at net.minecraft.client.main.Main.main(SourceFile:120)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at xv.e(SourceFile:108)
	at wv.k_(Entity.java:2622)
	at wv.d_(Entity.java:2362)
	at cpu.a(Render.java:377)
	at cqg.a(RenderItemFrame.java:228)
	at cqg.a(RenderItemFrame.java:83)
	at cqg.a(RenderItemFrame.java:255)

-- Entity being rendered --
Details:
	Entity Type: ItemFrame (adk)
	Entity ID: 2995377
	Entity Name: ~~ERROR~~ ClassCastException: amj cannot be cast to java.lang.String
	Entity's Exact location: 4128.50, 6.50, -572.97
	Entity's Block location: 4128.00,6.00,-573.00 - World: (4128,6,-573), Chunk: (at 0,0,3 in 258,-36; contains blocks 4128,0,-576 to 4143,255,-561), Region: (8,-2; contains chunks 256,-64 to 287,-33, blocks 4096,0,-1024 to 4607,255,-513)
	Entity's Momentum: 0.00, 0.00, 0.00
	Entity's Rider: ~~ERROR~~ NullPointerException: null
	Entity's Vehicle: ~~ERROR~~ NullPointerException: null

-- Renderer details --
Details:
	Assigned renderer: cqg@7e84a52b
	Location: 0.33,2.00,-0.94 - World: (0,2,-1), Chunk: (at 0,0,15 in 0,-1; contains blocks 0,0,-16 to 15,255,-1), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
	Rotation: 0.0
	Delta: 0.99863243
Stacktrace:
	at cpt.a(RenderManager.java:380)
	at cpt.a(RenderManager.java:337)
	at cpt.a(RenderManager.java:304)
	at ckn.a(RenderGlobal.java:654)
	at cji.a(EntityRenderer.java:1419)
	at cji.a(EntityRenderer.java:1333)

-- Affected level --
Details:
	Level name: MpServer
	All players: 5 total; [cio['ZombieKnifer223'/1144, l='MpServer', x=4128.17, y=4.50, z=-572.03], cip['PrincessAlisha'/2925897, l='MpServer', x=4129.84, y=4.00, z=-528.88], cip['leevipvp'/3010197, l='MpServer', x=4135.25, y=4.00, z=-541.19], cip['Chococat1199'/3020648, l='MpServer', x=4130.22, y=3.47, z=-527.28], cip['CombosFTW'/2937583, l='MpServer', x=4128.91, y=4.00, z=-526.81]]
	Chunk stats: MultiplayerChunkCache: 355, 355
	Level seed: 0
	Level generator: ID 01 - flat, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: -119.00,145.00,-133.00 - World: (-119,145,-133), Chunk: (at 9,9,11 in -8,-9; contains blocks -128,0,-144 to -113,255,-129), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 546417373 game time, 2335678808 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: ~~ERROR~~ ClassCastException: amj cannot be cast to java.lang.String
	Retry entities: 0 total; []
	Server brand: BungeeCord (git:BungeeCord-Bootstrap:1.8-SNAPSHOT:"6c4e684":1069) <- Spigot
	Server type: Non-integrated multiplayer server
Stacktrace:
	at cen.a(SourceFile:308)
	at bsu.b(Minecraft.java:2772)
	at bsu.a(Minecraft.java:443)
	at net.minecraft.client.main.Main.main(SourceFile:120)

-- System Details --
Details:
	Minecraft Version: 1.8
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 135847920 bytes (129 MB) / 303099904 bytes (289 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: AMD Radeon HD 6310 Graphics GL version 4.3.12618 Compatibility Profile Context 13.251.9001.1001, ATI Technologies Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Definitely; Client brand changed to 'Wurst'
	Type: Client (map_client.txt)
	Resource Packs: [faithful32pack_1_7_x.zip, TEAVSRP2.zip]
	Current Language: English (US)
	Profiler Position: N/A (disabled)
```

**Comment by GummiBoat** [comment]
Please put the Error into Markdown Code Block, by putting 3 ` in front and at the end of the error. And change the title to "FastBow crashes Wurst" or something similar.

**Comment by mcccs** [comment]
When I was buying a bow and arrows from an Admin Shop, it accidentally activate BowAinbot and when it shot, it crashed Wurst. Here is the report
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 7/27/15 1:05 PM
Description: Rendering entity in world

java.lang.ClassCastException: amj cannot be cast to java.lang.String
at xv.e(SourceFile:108)
at wv.k_(Entity.java:2622)
at wv.d_(Entity.java:2362)
at cpu.a(Render.java:377)
at cqg.a(RenderItemFrame.java:228)
at cqg.a(RenderItemFrame.java:83)
at cqg.a(RenderItemFrame.java:255)
at cpt.a(RenderManager.java:380)
at cpt.a(RenderManager.java:337)
at cpt.a(RenderManager.java:304)
at ckn.a(RenderGlobal.java:654)
at cji.a(EntityRenderer.java:1419)
at cji.a(EntityRenderer.java:1333)
at cji.b(EntityRenderer.java:1158)
at bsu.as(Minecraft.java:1144)
at bsu.a(Minecraft.java:421)
at net.minecraft.client.main.Main.main(SourceFile:120)

A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
at xv.e(SourceFile:108)
at wv.k_(Entity.java:2622)
at wv.d_(Entity.java:2362)
at cpu.a(Render.java:377)
at cqg.a(RenderItemFrame.java:228)
at cqg.a(RenderItemFrame.java:83)
at cqg.a(RenderItemFrame.java:255)

-- Entity being rendered --
Details:
Entity Type: ItemFrame (adk)
Entity ID: 2995377
Entity Name: ERROR ClassCastException: amj cannot be cast to java.lang.String
Entity's Exact location: 4128.50, 6.50, -572.97
Entity's Block location: 4128.00,6.00,-573.00 - World: (4128,6,-573), Chunk: (at 0,0,3 in 258,-36; contains blocks 4128,0,-576 to 4143,255,-561), Region: (8,-2; contains chunks 256,-64 to 287,-33, blocks 4096,0,-1024 to 4607,255,-513)
Entity's Momentum: 0.00, 0.00, 0.00
Entity's Rider: ERROR NullPointerException: null
Entity's Vehicle: ERROR NullPointerException: null

-- Renderer details --
Details:
Assigned renderer: cqg@7e84a52
Location: 0.33,2.00,-0.94 - World: (0,2,-1), Chunk: (at 0,0,15 in 0,-1; contains blocks 0,0,-16 to 15,255,-1), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
Rotation: 0.0
Delta: 0.99863243
Stacktrace:
at cpt.a(RenderManager.java:380)
at cpt.a(RenderManager.java:337)
at cpt.a(RenderManager.java:304)
at ckn.a(RenderGlobal.java:654)
at cji.a(EntityRenderer.java:1419)
at cji.a(EntityRenderer.java:1333)

-- Affected level --
Details:
Level name: MpServer
All players: 5 total; [cio['ZombieKnifer223'/1144, l='MpServer', x=4128.17, y=4.50, z=-572.03], cip['PrincessAlisha'/2925897, l='MpServer', x=4129.84, y=4.00, z=-528.88], cip['leevipvp'/3010197, l='MpServer', x=4135.25, y=4.00, z=-541.19], cip['Chococat1199'/3020648, l='MpServer', x=4130.22, y=3.47, z=-527.28], cip['CombosFTW'/2937583, l='MpServer', x=4128.91, y=4.00, z=-526.81]]
Chunk stats: MultiplayerChunkCache: 355, 355
Level seed: 0
Level generator: ID 01 - flat, ver 0. Features enabled: false
Level generator options: 
Level spawn location: -119.00,145.00,-133.00 - World: (-119,145,-133), Chunk: (at 9,9,11 in -8,-9; contains blocks -128,0,-144 to -113,255,-129), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
Level time: 546417373 game time, 2335678808 day time
Level dimension: 0
Level storage version: 0x00000 - Unknown?
Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Forced entities: ERROR ClassCastException: amj cannot be cast to java.lang.String
Retry entities: 0 total; []
Server brand: BungeeCord (git:BungeeCord-Bootstrap:1.8-SNAPSHOT:"6c4e684":1069) <- Spigot
Server type: Non-integrated multiplayer server
Stacktrace:
at cen.a(SourceFile:308)
at bsu.b(Minecraft.java:2772)
at bsu.a(Minecraft.java:443)
at net.minecraft.client.main.Main.main(SourceFile:120)

-- System Details --
Details:
Minecraft Version: 1.8
Operating System: Windows 8.1 (amd64) version 6.3
Java Version: 1.8.0_25, Oracle Corporation
Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
Memory: 135847920 bytes (129 MB) / 303099904 bytes (289 MB) up to 1060372480 bytes (1011 MB)
JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
Launched Version: Wurst
LWJGL: 2.9.1
OpenGL: AMD Radeon HD 6310 Graphics GL version 4.3.12618 Compatibility Profile Context 13.251.9001.1001, ATI Technologies Inc.
GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

Using VBOs: No
Is Modded: Definitely; Client brand changed to 'Wurst'
Type: Client (map_client.txt)
Resource Packs: [faithful32pack_1_7_x.zip, TEAVSRP2.zip]
Current Language: English (US)
Profiler Position: N/A (disabled)
```

**Comment by Alexander01998** [comment]
The `Wurst Client version` value is missing, that means you are using a very outdated version.
